### PR TITLE
Adding english and welsh locales

### PR DESCRIPTION
**Pull request by jongilbraith**
Waiting on a few more translations before moving forward.

### CODE REVIEW

**Comment by jongilbraith** [comment]
[Create functionality for user to be able to toggle between Welsh/English](https://trello.com/c/OmMohZm2/75-create-functionality-for-user-to-be-able-to-toggle-between-welsh-english)

File: config/routes.rb [hunk 1]
```diff
@@ -1,62 +1,63 @@
...
 
-    GuideRepository.slugs.each do |slug|
-      get slug,
+      get '/*id',
           controller: 'guides',
           action: 'show',
-          id: slug
-    end
+          as: :guide,
+          constraints: ->(req) { GuideRepository.slugs.include?(req.params[:id]) }
```
Comment (benlovell) [inline config/routes.rb]: Are `slugs` memoized? This is going to be super-expensive, per-request 🔥

File: config/routes.rb [hunk 2]
```diff
@@ -77,8 +78,13 @@
...
   end
 
+  delete '/locations_cache', to: 'locations_cache#destroy'
+
   if Rails.application.config.mount_javascript_test_routes
     mount JasmineRails::Engine => '/specs'
     mount JasmineFixtures => '/spec/javascripts/fixtures'
   end
+
+  get '/', to: redirect('/en')
```
Comment (benlovell) [inline config/routes.rb]: Mental note to do this in cloudflare so we can skip the hops to origin. ✏️ 🙉

File: config/routes.rb (hunk 1 above)
Comment (jongilbraith) [inline config/routes.rb]: [Yep](https://github.com/guidance-guarantee-programme/pension_guidance/blob/9b30c06846b21128e7100ba76f5903560f05a132/app/repositories/guide_repository.rb#L11)

**Comment by benlovell** [comment]
[As a customer I am able to view the welsh content for particular pages](https://trello.com/c/haP0OijL/117-as-a-customer-i-am-able-to-view-the-welsh-content-for-particular-pages)

**Pull request merged**